docs/src/optimization.md

```diff
@@ -143,17 +143,14 @@ The $\theta$ vector is
 MixedModels.getθ(t21)
 ```
 
-Random-effects terms in the model formula that have the same grouping factor are amagamated into a single `ReMat` object.
+Random-effects terms in the model formula that have the same grouping factor are amalgamated into a single `ReMat` object.
 ```@example Main
-fm3 = fit!(zerocorr!(LinearMixedModel(@formula(reaction ~ 1+days+(1+days|subj)),
-    sleepstudy)))
+fm3 = fit!(LinearMixedModel(@formula(reaction ~ 1+days+(1|subj) + (0+days|subj)),
```

Comment:
I've left this here because we have an example without `0+` above and this hows that both variants are valid.

Reply:
Quite reasonable.